inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -383,6 +384,7 @@ ModelPtr FrontEnd::runCommonPasses(ie::ICNNNetwork& network, const UnsupportedLa
             // Disable shape inference (WA for generic operations)
             ngraph::op::GenericIE::DisableReshape noReshape(nGraphFunc);
 
+            ngraph::pass::CommonOptimizations().run_on_function(nGraphFunc);
```

Comment:
Does it mean for MYX it called 2 times: inside `LoadExeNetworkImpl` and here?
I think it should be placed in `LoadExeNetworkImpl` of HDDL plugin instead